docs/user_manual/site_configuration.md

```diff
@@ -30,30 +24,22 @@ groups, but they should be able to help you get set up.
 If you require a new account, password reset, or are otherwise unable to log in to MISO, you should talk to your MISO
 administrators.
 
-{% include userman-toplink.md %}
-
-
 
-{% assign sub = sub | plus: 1 %}
-{% include userman-heading.md section=page.section sub=sub title="Issue Trackers" %}
+## Issue Trackers
 
 MISO integrates with issue trackers in order to display and link to issues from within MISO. The only
 currently-supported issue tracker is JIRA. If JIRA integration is configured for your site, you will see a list of
 related issues on the Project page and the Run page. These lists are described in more detail in the
-[Projects](projects.html) and [Sequencer Runs](sequencer_runs.html) sections.
+[Projects](../projects/) and [Sequencer Runs](../sequencer_runs/) sections.
+
 
-{% assign figure = 1 %}
 MISO may also be configured to take bug reports. If it is, a "Report a problem" link will appear near the top right of
 the screen (see figure {{figure}}). Click this link to report a bug in your issue tracker.
```

Comment:
The `(see figure {{figure}})` text in this file is rendering literally, instead of replacing the `figure` variable.

Reply:
Fixed.